ownCloud/Viewer/ImageDisplayViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  ImageDisplayViewController.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 17/10/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+import UIKit
+import ownCloudSDK
+
+class ImageDisplayViewController : DisplayViewController {
+
+	var scrollView: UIScrollView?
+	var imageView: UIImageView?
+
+	override func renderSpecificView() {
+		scrollView = UIScrollView()
+
+		scrollView?.translatesAutoresizingMaskIntoConstraints = false
+		scrollView?.maximumZoomScale = 6.0
+		scrollView?.bounces = false
+		scrollView?.bouncesZoom = false
+		scrollView?.delegate = self
+		self.view.addSubview(scrollView!)
+
+		NSLayoutConstraint.activate([
+			scrollView!.leftAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leftAnchor),
+			scrollView!.rightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.rightAnchor),
+			scrollView!.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
+			scrollView!.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor)
```

Comment:
By using the `safeAreaLayoutGuide`, the scroll view does only fill part of the available space and is "cut off" on the bottom on iPhone X:
<img width="545" alt="bildschirmfoto 2018-10-17 um 23 13 40" src="https://user-images.githubusercontent.com/639669/47116575-4a4d2d80-d262-11e8-9315-e322302cc521.png">